### PR TITLE
adding nanoserver image

### PR DESCRIPTION
**Pull request by luthermonson**
https://github.com/rancher/rancher/issues/31031

### CODE REVIEW

File: package/Dockerfile-windows.agent [hunk 1]
```diff
@@ -25,4 +25,8 @@ RUN $URL = 'https://{0}/fleet/{1}/fleetagent-windows-amd64.exe' -f $env:RELEASES
     Write-Host ('Downloading dapper from {0} ...' -f $URL); \
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
     Invoke-WebRequest -UseBasicParsing -OutFile c:/fleet/fleetagent-windows.exe -Uri $URL;
+FROM mcr.microsoft.com/windows/nanoserver:${SERVERCORE_VERSION}
+USER ContainerAdministrator
+COPY --from=builder /fleet /fleet
+COPY --from=builder /git /git
 CMD ["C:\fleet\fleetagent-windows.exe"]
```
Comment (nickgerace) [inline package/Dockerfile-windows.agent:32]: I believe we have to copy the `agent` as well. Three copies?

**Review by nickgerace — CHANGES_REQUESTED** [review]

**Comment by nickgerace** [comment]
**Note**: this change is pending successful RC of Fleet Windows.

File: package/Dockerfile-windows.agent (hunk 1 above)
Comment (luthermonson) [reply package/Dockerfile-windows.agent:32]: its not there

Comment (nickgerace) [reply package/Dockerfile-windows.agent:32]: Miscommunication: I though `/fleet` was a binary copy for some reason. It's a directory copy and contains the `agent`. Resolving this.

**Comment by nickgerace** [comment]
We should consider this now @luthermonson.

File: package/Dockerfile-windows.agent (hunk 1 above)
Comment (nickgerace) [inline package/Dockerfile-windows.agent:32]: This requires `\\` rather than `\`.

**Comment by nickgerace** [comment]
Closing since stale. We should make a new PR once we start pursuing again: https://github.com/rancher/windows/issues/78

**Pull request closed without merging**